jupyter-book/annotation/annotation.yml

```diff
@@ -4,10 +4,10 @@ dependencies:
   - python=3.7
   - pip
   - pip:
-     - jupyterlab
-     - scanpy==1.9
-     - umap-learn>=0.5
-     - pynndescent
-     - nb-black # for pretty code formatting
-     - leidenalg
-     - celltypist
+      - jupyterlab
+      - scanpy==1.9
+      - umap-learn>=0.5
+      - pynndescent
+      - nb-black # for pretty code formatting
```

Comment:
Don't think that users trying to run the notebook will need nb-black. That's more of a convenience for you. Funfact: the pre-commit setup runs black on Jupyter notebook anyways.